src/Symfony/Component/Notifier/Bridge/OvhCloud/OvhCloudTransportFactory.php

```diff
@@ -29,7 +30,17 @@ public function create(Dsn $dsn): TransportInterface
         $applicationKey = $this->getUser($dsn);
         $applicationSecret = $this->getPassword($dsn);
         $consumerKey = $dsn->getOption('consumer_key');
+
+        if (!$consumerKey) {
+            throw new IncompleteDsnException('Missing consumer_key.', $dsn->getOriginalDsn());
+        }
+
         $serviceName = $dsn->getOption('service_name');
+
+        if (!$serviceName) {
+            throw new IncompleteDsnException('Missing service_name.', $dsn->getOriginalDsn());
+        }
+
```

Comment:
This should be considered a bugfix

Reply:
see https://github.com/symfony/symfony/pull/39406#discussion_r539307754